### PR TITLE
Added comment in index.md

### PR DESCRIPTION
**Pull request by Abhishek01039**
Fixes https://github.com/flutter/website/issues/5074.

### CODE REVIEW

File: src/docs/index.md [hunk 1]
```diff
@@ -92,6 +92,7 @@ You might also find these docs useful:
 * [State management][]
 * [Animations][]
 
+The documentation follows the latest stable version of flutter
```
Comment (sfshaza2) [inline src/docs/index.md:95]: Sorry to be so nit-picky, but please make the f in Flutter uppercase. Also, add a period at the end. Thx!

**Comment by Hixie** [comment]
maybe "refers to" rather than "follows"? But in general this is fine by me. @sfshaza2 the context here is that it turns out we don't seem to actually say anywhere that the website is intended to document the latest stable, rather than, say, the latest dev build, and it would be useful to have something to point to when confusion about this arises.

**Comment by sfshaza2** [comment]
Yes, I think adding this comment (or a version thereof) is an excellent idea. Lately I've been barraged by PRs where some eager beavers (whom we love) are trying to update code/text on flutter.dev to post 1.22 changes/deprecations/etc.  I've repeatedly had to explain that we document against stable.

I just want Flutter to be upper case and for the sentence to end in a period.  :). Also, a more accurate line might be:

The documentation on this site reflects the latest stable release of Flutter.

or

Unless otherwise noted, the code on this website is tested against the latest stable release of Flutter.

**Comment by Abhishek01039** [comment]
```The documentation on this site reflects the latest stable release of Flutter.```
This seems fine.
should I add this @sfshaza2 ?

**Review by sfshaza2 — APPROVED** [review]
LGTM

**Pull request merged**